src/main/java/io/confluent/kafkarest/KafkaRestContextImpl.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Shared, global state for the REST proxy server, including configuration and connection pools.
  */
-public class Context {
+public class KafkaRestContextImpl implements KafkaRestContext {
```

Comment:
SimpleKafkaRestContext could be a confusing as there are Simple* classes that define something else. (ex. SimpleConsumerManager, SimpleConsumerPool etc). I would go for  DefaultKafkaRestContext here